src/vignette/storage/s3.clj

```diff
@@ -105,6 +105,8 @@
   (put-object [this resource bucket path]
     (let [file (file-stream resource)
           mime-type (content-type resource)]
+      (if (perf/timing :s3-get (not (s3/bucket-exists? creds bucket)))
```

Comment:
I wouldn't time bucket read/creation in the same measurement as file reads/writes